src/Services/EncounterService.php

```diff
@@ -201,6 +202,8 @@ class.notes as class_title,
                                class_code,
                                facility_id,
                                discharge_disposition,
+                               encounter_type_code,
+                               encounter_type_description,
```

Comment:
@adunsulag check me here.

Reply:
@sjpadgett You'll need to add encounter_type_code & encounter_type_description to the outer select on line 157 if you want to actually see this data.  Because of the way searching is done and how we have duplicate columns for 'pid' on the joins on everything we have the multi-select.  This is where I wish we and an ORM.